includes/admin/post-types/post-tables/class.llms.admin.post.table.lessons.php

```diff
@@ -109,6 +120,112 @@ public function manage_columns( $column, $post_id ) {
 		}// End switch().
 
 	}
+	/**
+	 * Add  filters
+	 * 
+	 * @return string/html
+	 * @since 3.9.6
+	 */
+	 public function filters($post_type){
+		
+		//only add filter to post type you want
+		if ('lesson' == $post_type){
```

Comment:
reduce complexity with `if ( 'lesson' !== $post_type ) { return; }` rather than wrapping the whole function in an if